README.rst

```diff
@@ -324,92 +324,6 @@ in the process of being migrated to KWIVER and will leave this repository soon.
 Continued support is not guaranteed and behavior may diverge from documentation.
```

Comment:
We should remove this paragraph as well.  And the section heading above it.  and remove tools from the directory list on line 315.